commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -156,6 +156,10 @@
  </ul>
  -->
  -->
+ <h3 class=released-version id="v1_63_0">1.63.0 (03.05.2021)</h3>
+ <ul>
+ <li class=fixed-in-release>Fixed bug in the {@link io.sphere.sdk.retry.RetryAction#ofScheduledRetry(long, Function)} introduced try catch to handle the throwing of an exception
```

Comment:
```suggestion
 <li class=fixed-in-release>Fixed deadlock bug in the {@link io.sphere.sdk.retry.RetryAction#ofScheduledRetry(long, Function)} in case of exception
```